### ISSUE TITLE
[Accessibility][Textfield]: Component should have aria-disabled attribute set when it is disabled

### ISSUE THREAD

**Issue by prmittal** [OfficeDev/office-ui-fabric-react#1083]
For disabled textfield, aria-disabled attribute should be specified, else screen readers will have trouble reading it clearly. See this: http://www.w3.org/TR/wai-aria/states_and_properties#aria-disabled

Repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/textfield

The html for disabled textfield is as below:
`<div class="ms-TextField is-disabled"><label for="TextField1785" class="ms-Label">Disabled TextField</label><input type="text" label="Disabled TextField" disabled="" id="TextField1785" value="" class="ms-TextField-field" aria-describedby="TextFieldDescription1786" aria-invalid="false"><!-- react-text: 29752 --><!-- /react-text --><!-- react-text: 29753 --><!-- /react-text --></div>`

**Comment by mdahamiwal** [comment]
Same as #1084, disabled components are not supposed to be focusable, we shouldn't need aria-disabled on these?

**Comment by satishc2437** [comment]
Please checkout my reply in #1084

**Comment by joschect** [comment]
This looks like it was fixed in #1236 . I'm closing it for now, but please feel free to reopen it if that's not the case.
